src/main/java/net/fabricmc/tinyremapper/extension/mixin/soft/annotation/injection/CommonInjectionAnnotationVisitor.java

```diff
@@ -175,6 +178,24 @@ public void visit(String name, Object value) {
 						super.visit(name, value);
 					}
 				};
+			} else if (remap && name.equals(AnnotationElement.TARGET)) {	// All
+				return new AnnotationVisitor(Constant.ASM_VERSION, av) {
+					@Override
+					public AnnotationVisitor visitAnnotation(String name, String descriptor) {
+						if (!descriptor.equals(Annotation.DESC)) {
+							throw new RuntimeException("Unexpected annotation " + descriptor);
+						}
+
+						MemberType expectedDescType;
+						if (annotationDesc.equals(Annotation.MODIFY_CONSTANT)) {
```

Comment:
This does not pass our checkstyle rules, causing the build to fail.

Reply:
Updated. Completely forgot about the checkstyle before pushing